pkg/kritis/review/review_test.go

```diff
@@ -0,0 +1,298 @@
+		shdErr            bool
+	}{
+		{
+			name:              "vulnz w attestation for Webhook shd not handle voilations",
+			image:             vulnImage,
+			isWebhook:         true,
+			attestations:      validAtts,
+			handledViolations: 0,
+			isAttested:        true,
+			shdAttestImage:    false,
+			shdErr:            false,
+		},
+		{
+			name:              "vulnz w/o attestation for Webhook shd handle voilations",
+			image:             vulnImage,
+			isWebhook:         true,
+			attestations:      []metadata.PGPAttestation{},
+			handledViolations: 1,
+			isAttested:        false,
+			shdAttestImage:    false,
+			shdErr:            true,
+		},
+		{
+			name:              "no vulnz w/o attestation for webhook shd add attestation",
+			image:             noVulnImage,
+			isWebhook:         true,
+			attestations:      []metadata.PGPAttestation{},
+			handledViolations: 0,
+			isAttested:        false,
+			shdAttestImage:    true,
+			shdErr:            false,
+		},
+		{
+			name:              "vulnz w attestation for cron shd handle vuln",
+			image:             vulnImage,
+			isWebhook:         false,
+			attestations:      validAtts,
+			handledViolations: 1,
+			isAttested:        true,
+			shdAttestImage:    false,
+			shdErr:            true,
+		},
+		{
+			name:              "vulnz w/o attestation for cron shd handle vuln",
+			image:             vulnImage,
+			isWebhook:         false,
+			attestations:      []metadata.PGPAttestation{},
+			handledViolations: 1,
+			isAttested:        false,
+			shdAttestImage:    false,
+			shdErr:            true,
+		},
+		{
+			name:              "no vulnz w/o attestation for cron shd verify attestations",
+			image:             noVulnImage,
+			isWebhook:         false,
+			attestations:      []metadata.PGPAttestation{},
+			handledViolations: 0,
+			isAttested:        false,
+			shdAttestImage:    false,
+			shdErr:            false,
+		},
+		{
+			name:              "no vulnz w attestation for cron shd verify attestations",
+			image:             noVulnImage,
+			isWebhook:         false,
+			attestations:      []metadata.PGPAttestation{{Signature: sigNoVuln, KeyId: "sec"}},
+			handledViolations: 0,
+			isAttested:        true,
+			shdAttestImage:    false,
+			shdErr:            false,
+		},
+	}
+	for _, tc := range tests {
+		th := violation.MemoryStrategy{
+			Violations:   map[string]bool{},
+			Attestations: map[string]bool{},
+		}
+		t.Run(tc.name, func(t *testing.T) {
+			cMock := &testutil.MockMetadataClient{
+				PGPAttestations: tc.attestations,
+			}
+			r := New(cMock, &th, tc.isWebhook, &Config{
+				validate: testValidate,
+				secret:   sMock,
+			})
+			if err := r.Review([]string{tc.image}, isps, nil); (err != nil) != tc.shdErr {
+				t.Fatalf("expected review to return error %t, actual error %s", tc.shdErr, err)
+			}
+			if len(th.Violations) != tc.handledViolations {
+				t.Fatalf("expected to handle %d violations. Got %d", tc.handledViolations, len(th.Violations))
```

Comment:
"got" before "expected" (here and elsewhere)

Reply:
i think all our tests and helper do the "expected" before "got".
https://github.com/grafeas/kritis/blob/master/pkg/kritis/testutil/util.go#L36

How about we switch it on PR later.